Android/FirebaseJobDispatcher/nuget/Xamarin.Firebase.JobDispatcher.nuspec

```diff
@@ -11,6 +11,9 @@
     <copyright>Copyright (c) 2017 Xamarin Inc</copyright>
     <projectUrl>https://github.com/xamarin/XamarinComponents/tree/master/Android/FirebaseJobDispatcher</projectUrl>
     <licenseUrl>https://github.com/xamarin/XamarinComponents/blob/master/Android/FirebaseJobDispatcher/License.md</licenseUrl>
+    <dependencies>
+        <dependency id="Xamarin.Android.Support.v4" version="25.3.1" />
```

Comment:
Shouldn't `Xamarin.GooglePlayServices.Gcm` be listed as a dependency too? or is it not strictly _required_